testnet/stacks-node/src/config.rs

```diff
@@ -748,7 +748,7 @@ impl BurnchainConfig {
             local_mining_public_key: None,
             burnchain_op_tx_fee: MINIMUM_DUST_FEE,
             process_exit_at_block_height: None,
-            poll_time_secs: 30, // TODO: this is a testnet specific value.
+            poll_time_secs: 10, // TODO: this is a testnet specific value.
```

Comment:
Is this a default we want to change for all testnet nodes, or is this a config change that only makes sense for local testing?

Reply:
I'll revert that.  In general, this value has to be less than the block time (and I was testing with a 30s block time)

Reply:
I thought we changed this to 30s because our testnet bitcoind was getting swamped?

Reply:
If our block time is now two minutes, then 30s should be fine.  I'll revert it.